scripts/create_adaptor_packaging_artifact.sh

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+set -xeou pipefail
+
+APP_LOWERCASE=keyshot
+APP_PASCAL_CASE=KeyShot
+ADAPTOR_NAME=deadline-cloud-for-$APP_LOWERCASE
+
+# This script generates an tar.gz artifact from $ADAPTOR_NAME and its dependencies
+# that can be used to create a package for running the adaptor.
+
+SCRIPTDIR=$(realpath $(dirname $0))
+
+SOURCE=0
+# Python 3.11 is for https://vfxplatform.com/ CY2024
+PYTHON_VERSION=3.11
+CONDA_PLATFORM=win-64
+TAR_BASE=
+
+while [ $# -gt 0 ]; do
+    case "${1}" in
+    --source) SOURCE=1 ; shift ;;
+    --platform) CONDA_PLATFORM="$2" ; shift 2 ;;
+    --python) PYTHON_VERSION="$2" ; shift 2 ;;
+    --tar-base) TAR_BASE="$2" ; shift 2 ;;
+    *) echo "Unexpected option: $1"; exit 1 ;;
+  esac
+done
+
+if [ "$CONDA_PLATFORM" = "linux-64" ]; then
+    PYPI_PLATFORM=manylinux2014_x86_64
```

Comment:
How did you choose this platform and the Mac on? They look outdated but I'd guess it's required for some reason.

Reply:
Here's a list of the manylinux options: https://github.com/pypa/manylinux
I couldn't find docs for the mac platforms

Reply:
I used the same platforms as all the other packaging artifact files; not sure exactly which this was chosen for them but I wanted to stay consistent